src/component/vue-hcaptcha.vue

```diff
@@ -4,9 +4,13 @@
   </div>
 </template>
 <script>
-const CaptchaScript = (cb) => {
+const CaptchaScript = (cb, choosenlang) => {
+    let lang = '';
+    if(typeof lang !== 'undefined') { // Set the selected language
```

Comment:
I think you want `choosenlang` here, but really the simplest method is to do the following:
```js
script.async = true;
script.src = "https://hcaptcha.com/1/api.js?render=explicit"
if (choosenlang) 
    script.src += `&hl=${choosenlang}`;
```